Sources/xcproj/XCSharedData.swift

```diff
@@ -27,6 +32,7 @@ final public class XCSharedData {
         }
         self.schemes = path.glob("xcschemes/*.xcscheme")
             .flatMap { try? XCScheme(path: $0) }
+        self.breakpoints = try? XCBreakpointList(path: path + "xcdebugger/Breakpoints_v2.xcbkptlist")
```

Comment:
Out of curiosity, is the name always `Breakpoints_v2.xcbkptlist`?

Reply:
Yes, I also found it curious, but it is always that file and all breakpoints go there.
And since old versions of Xcode, it's v2 specification.

Reply:
Cool 👍